[Bug, mathutil] AttributeError: 'Series' object has no attribute 'has_nulls'. Did you mean: 'is_null'?

To reproduce:
```console
pdr_backend/util/test_noganache/test_mathutil.py::test_has_nan__or_None
```

Returns:
```text
________________________________________________________________________________________________ test_has_nan__or_None _________________________________________________________________________________________________

    @enforce_types
    def test_has_nan__or_None():
        # 1d array
        assert not has_nan(np.array([1.0, 2.0, 3.0, 4.0]))
        assert has_nan(np.array([1.0, 2.0, np.nan, 4.0]))
        assert has_nan(np.array([1.0, None, 3.0, 4.0]))
        assert has_nan(np.array([1.0, None, np.nan, 4.0]))
    
        # 2d array
        assert not has_nan(np.array([[1.0, 2.0], [3.0, 4.0]]))
        assert has_nan(np.array([[1.0, 2.0], [np.nan, 4.0]]))
        assert has_nan(np.array([[1.0, None], [3.0, 4.0]]))
        assert has_nan(np.array([[1.0, None], [np.nan, 4.0]]))
    
        # pd Series
        assert not has_nan(pd.Series([1.0, 2.0, 3.0, 4.0]))
        assert has_nan(pd.Series([1.0, 2.0, np.nan, 4.0]))
        assert has_nan(pd.Series([1.0, None, 3.0, 4.0]))
        assert has_nan(pd.Series([1.0, None, np.nan, 4.0]))
    
        # pd DataFrame
        assert not has_nan(pd.DataFrame({"A": [1.0, 2.0], "B": [3.0, 4.0]}))
        assert has_nan(pd.DataFrame({"A": [1.0, 2.0], "B": [np.nan, 4.0]}))
        assert has_nan(pd.DataFrame({"A": [1.0, None], "B": [3.0, 4.0]}))
        assert has_nan(pd.DataFrame({"A": [1.0, None], "B": [np.nan, 4.0]}))
    
        # pl Series
>       assert not has_nan(pl.Series([1.0, 2.0, 3.0, 4.0]))

pdr_backend/util/test_noganache/test_mathutil.py:125: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
venv/lib/python3.12/site-packages/enforce_typing/decorator.py:29: in wrapper
    return func(*args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

x = shape: (4,)
Series: '' [f64]
[
	1.0
	2.0
	3.0
	4.0
]

    @enforce_types
    def has_nan(
        x: Union[np.ndarray, pd.DataFrame, pd.Series, pl.DataFrame, pl.Series]
    ) -> bool:
        """Returns True if any entry in x has a nan _or_ a None"""
        if isinstance(x, np.ndarray):
            has_None = (x == None).any()  # pylint: disable=singleton-comparison
            return has_None or np.isnan(np.min(x))
    
        if isinstance(x, pl.Series):
>           has_None = x.has_nulls()  # type: ignore[attr-defined]
E           AttributeError: 'Series' object has no attribute 'has_nulls'. Did you mean: 'is_null'?
```

--- a/pdr_backend/util/mathutil.py
+++ b/pdr_backend/util/mathutil.py
@@ -55,11 +55,11 @@ def has_nan(
         return has_None or np.isnan(np.min(x))
 
     if isinstance(x, pl.Series):
-        has_None = x.has_nulls()
+        has_None = x.is_null().any()
         return has_None or sum(x.is_nan()) > 0  # type: ignore[union-attr]
 
     if isinstance(x, pl.DataFrame):
-        has_None = any(col.has_nulls() for col in x)
+        has_None = any(col.is_null().any() for col in x)
         return has_None or sum(sum(x).is_nan()) > 0  # type: ignore[union-attr]
 
     # pd.Series or pd.DataFrame

